org.eclipse.corrosion.feature/feature.xml

```diff
@@ -19,6 +19,10 @@
       %license
    </license>
 
+   <requires>
+      <import plugin="org.eclipse.cdt.core.native" version="0.0.0"/>
```

Comment:
Is it the only bundle in the org.eclipse.cdt.native feature? It would seem safer to require the whole feature instead of a single bundle, so the specific fragments are installed too (which may be missing here)

Reply:
Done